packages/react-magma-dom/src/components/AlertBase/index.tsx

```diff
@@ -157,33 +205,29 @@ const StyledAlert = styled.div<AlertBaseProps>`
     text-decoration: underline;
 
     &:focus {
-      outline: 2px solid ${props =>
-        props.variant === 'warning'
-          ? props.theme.colors.focus
-          : props.theme.colors.focusInverse};
-      }
+      outline: 2px solid
+        ${props =>
+          props.variant === 'warning'
+            ? props.theme.colors.focus
+            : props.theme.colors.focusInverse};
     }
   }
 `;
 
 const StyledAlertInner = styled.div<AlertBaseProps>`
-  background-color: ${props => buildAlertBackground(props)};
+  background: ${buildAlertBackground};
+  border: 1px solid ${buildAlertBorder};
   border-radius: ${props => props.theme.borderRadius};
-  color: ${props =>
-    props.variant === AlertVariant.muted
-      ? props.theme.colors.neutral03
-      : props.isInverse
-      ? props.theme.colors.neutral08
-      : props.theme.colors.neutral};
+  color: ${buildAlertColor};
   display: flex;
   position: relative;
-  padding-right: 12px; 
 
   ${props =>
     props.isToast &&
     css`
-      border: 1px solid ${props.theme.colors.neutral08};
-      box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.4);
+      box-shadow: ${props.isInverse
+        ? `0 2px 8px 0 rgba(0, 0, 0, 0.7)`
+        : `0 2px 8px 0 rgba(0, 0, 0, 0.4)`};
```

Comment:
Since we are using `transparentize` in most other places, I wonder if we should use that here too for consistency?

Reply:
I'm not personally familiar with `transparentize` so I'll defer to you all for how best to implement the `rgba` values.